client/src/app/pages/home/components/WatchedSbom.tsx

```diff
@@ -61,7 +63,23 @@ export const WatchedSbom: React.FC<WatchedSbomProps> = ({
     sbom: currentSbom,
     isFetching: isFetchingCurrentSbom,
     fetchError: fetchErrorCurrentSbom,
-  } = useFetchSBOMById(sbomId ?? undefined);
+  } = useFetchSBOMById(sbomId ?? undefined, undefined, 1);
+
+  // Handle SBOMs that no longer exist
```

Comment:
Don't think that comment is needed.
But this is just me not being a big fan of comments.